src/components/InvertedFlatList/BaseInvertedFlatList/index.e2e.tsx

```diff
@@ -23,15 +23,15 @@ function BaseInvertedFlatListE2e(props: BaseInvertedFlatListProps, ref: React.Fo
     const maintainVisibleContentPosition = useMemo(() => {
         const config: ScrollViewProps['maintainVisibleContentPosition'] = {
             // This needs to be 1 to avoid using loading views as anchors.
-            minIndexForVisible: 1,
+            minIndexForVisible: (rest.data?.length ?? 0) === 0 ? 0 : Math.min(1, (rest.data?.length ?? 0) - 1),
```

Comment:
```suggestion
            minIndexForVisible: (rest.data?.length ?? 0) === 0 ? 0 : Math.min(1, rest.data.length - 1),
```

I think this should work? since if there is no `rest.data?.length` then we'll just use 0 and won't get to this condition